builtin/logical/database/path_roles.go

```diff
@@ -504,17 +540,20 @@ func (b *databaseBackend) pathStaticRoleCreateUpdate(ctx context.Context, req *l
 		if err := req.Storage.Put(ctx, entry); err != nil {
 			return nil, err
 		}
+		item, err = b.popFromRotationQueueByKey(name)
+		if err != nil || item == nil {
```

Comment:
It looks like `popFromRotationQueueByKey` cannot return a nil `item` and a nil `error` at the same time, and if `item` is nil it would return a `queue.ErrEmpty` error.
```suggestion
		if err != nil {
```